system/autoware_default_adapi/src/planning.cpp

```diff
@@ -49,16 +78,121 @@ std::vector<typename rclcpp::Subscription<T>::SharedPtr> init_factors(
 }
 
 template <class T>
-T merge_factors(const rclcpp::Time stamp, const std::vector<typename T::ConstSharedPtr> & factors)
+std::vector<T> convert(const std::vector<PlanningFactor> & factors)
```

Comment:
clang-tidy is throwing the following error: 
> /__w/autoware.universe/autoware.universe/system/autoware_default_adapi/src/planning.cpp:81:60: error: unused parameter 'factors' [clang-diagnostic-unused-parameter]
std::vector<T> convert(const std::vector<PlanningFactor> & factors)


```suggestion
std::vector<T> convert([[maybe_unused]] const std::vector<PlanningFactor> & factors)
```